api/src/models/JobMetadata.ts

```diff
@@ -58,31 +60,61 @@ class JobMetadata {
     }
 
     get derivativeStatus(): Record<string, unknown> {
-        // TODO: populate with real data
+        let lockfileExists: boolean = fs.existsSync(this.derivativeLockfile);
```

Comment:
Should this be const since it is never changed?

Reply:
Do we want a lint action on PRs?

Reply:
If there's an easy way to wire it up, it wouldn't hurt.

Reply:
Not sure if we want to do CI stuff in Geoff's personal repo, though; maybe we should think about moving this under FalveyLibraryTechnology soon.